### DISCUSSION TITLE
Needs correct documentation

### DISCUSSION THREAD

**Discussion by SanjayRam07** [maplibre/maplibre-native#2193] [Q&A]
hello guys, i'm a new comer. I want to work with maplibre and checked docs. But there are so many docs with different steps in each of them. Kindly provide me a doc explaining how to use map libre in android.

**Comment by louwers** [comment]
Here is our getting started guide for Android https://maplibre.org/maplibre-native/docs/book/android/index.html

I recommend using `implementation 'org.maplibre.gl:android-sdk:11.0.0-pre4'` since 11.0.0 will be released at the end of this month.
